### ISSUE TITLE
Batch size used for different datasets

### ISSUE THREAD

**Issue by impiga** [open-mmlab/mmsegmentation#198]
Hi, thanks for your great work.

I notice this note in the [document](https://github.com/open-mmlab/mmsegmentation/blob/master/docs/getting_started.md#train-a-model):
> **Important**: The default learning rate in config files is for 4 GPUs and 2 img/gpu (batch size = 4x2 = 8). Equivalently, you may also use 8 GPUs and 1 imgs/gpu since all models using cross-GPU SyncBN.

However, in [configs/\_base\_/datasets](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/_base_/datasets), I find that default settings for ade20k/cityscapes/pascal voc are 4 img/gpu, 2 img/gpu and 4 img/gpu respectively.

My question: How many gpus and how many images per gpu are used to train the provided checkpoints? For ade20k and pascal voc, if 4 gpus are used, the total batch size will be 16 and this conflict with the note above. Could you help to figure out the default setting?

I really appreciate the help.

**Comment by xvjiarui** [comment]
Hi @impiga 
Sorry for the confusion. We use 4(gpus)x2(imgs) for cityscapes and 4x4 for ade20k and pascal voc.

**Comment by impiga** [comment]
OK. Thanks for your quick reply(#^.^#).
